envs/example/defaults.yml

```diff
@@ -338,6 +338,54 @@ keystone:
       internal_url: https://{{ endpoints.main }}:6384
       admin_url: https://{{ endpoints.main }}:6384
 
+openldap:
+  enabled: True
+
+  # openldap server settings
+  url: ldap://{{ undercloud_floating_ip }}:390
+  dc: "example"
+  suffix: "dc=example,dc=org"
+  root_cn: "admin"
+  root_dn: "cn=admin,dc=example,dc=org"
+  company_ou: "openstack"
+  organizationalUnit: "ou=openstack,dc=example,dc=org"
+  users_ou: "users"
+  groups_ou: "groups"
+  users_dn: "ou=users,ou=openstack,dc=example,dc=org"
+  groups_dn: "ou=groups,ou=openstack,dc=example,dc=org"
+
+  password_policy_enabled: False
+  password_policy_ou: "policies"
+  password_policy_ou_dn: "ou=policies,ou=openstack,dc=example,dc=org"
+  password_policy_cn: "default"
+  password_policy_dn: "cn=default,ou=policies,ou=openstack,dc=example,dc=org"
```

Comment:
I checked this again and jinja/ansible is not able to template recursive defiistions in a namespace, top levels definitions are fine. 

for example the below fails as the dictionary openldap is not yet created when we try to refer to openldap.root_cn.
openldap:
&nbsp;&nbsp;root_cn: "admin"
&nbsp;&nbsp;root_dn: "cn={{openldap.root_cn}},dc=example,dc=org"